woocommerce/wc-admin
# Add 'Clear' button to Search input

Follow-up of #1381.

Adds a _Clear_ button to table search input.

### Accessibility


### Screenshots
![search-2](https://user-images.githubusercontent.com/3616980/51929817-6a5fc180-23f9-11e9-8f5e-f2d81d4d8070.gif)

### Detailed test instructions:
- Go to any report with a table search input.
- Make a search and verify a _Clear_ button appears.
- Press it and verify it removes the search terms.
- Add an advanced filter and verify the _Clear_ button doesn't appear when you add a term.
